tests/test_scene_rendering/test_cli_flags.py

```diff
@@ -164,3 +164,27 @@ def test_custom_folders(tmp_path, manim_cfg_file, simple_scenes_path):
 
     exists = (tmp_path / "SquareToCircle.png").exists()
     assert exists, "--custom_folders did not produce the output file"
+
+
+@pytest.mark.slow
+def test_dash_as_filename(tmp_path):
+    code = "class Test(Scene):\n    def construct(self):\n        self.add(Circle())\n        self.wait(1)"
+    command = " ".join(
+        [
+            "echo",
+            f'"{code}"',
+            "|",
```

Comment:
I think this should be
```suggestion
            "||",
```

Reply:
No, the piping operator is one `|`. The double `||` is an "or" operator, so if the echo succeeds, then the next command wouldn't be run